lib/tasks/scheme_ten.rake

```diff
@@ -0,0 +1,19 @@
+namespace :scheme_ten do
```

Comment:
not a blocker. I'd namespace this to db with task name, or sub-namespace, of :seed
```
namespace :db
  namespace :scheme_ten
    namespace :seed
```

and use

```
rake db:scheme_ten:seed
```
or alternative
```
rake db:seed:scheme_ten
```
others could be namepaced therein in future

Reply:
rails adds a `db:seed` task so the alternative above extends that notion above